schedule/migration/aarch64_regression_test_offline.yaml

```diff
@@ -44,6 +44,7 @@ schedule:
   - console/prepare_test_data
   - console/consoletest_setup
   - '{{check_upgraded_service}}'
+  - console/kdump_and_crash
```

Comment:
It's better to move this after - '{{regression_tests}}', WDYT?

Reply:
changed. thanks

Reply:
now moved after orphan_package_check, cause if we moved after - '{{regression_tests}}' it'll run
after consoletest_finish which looks weird.

Reply:
Why not insert this to before consoletest_finish in 'regression_tests'?

Reply:
yes. it's in the regression_tests before consoletest_finish.